README.md

```diff
@@ -30,8 +30,8 @@ Configuration of the application is managed using [TOML](https://toml.io). The a
 [database]
   host     = "localhost"
   port     = 5432
-  dbname   = "tagger"
-  user     = "user"
+  dbname   = "tagger-db"
```

Comment:
I don't want to nitpick on this, but now it feels kind of strange to have `dbname` and `user` with a `tagger` prefix, while `password` does not have it